lxqt-admin-user/mainwindow.cpp

```diff
@@ -76,6 +76,17 @@ void MainWindow::reloadUsers()
     }
 }
 
+void MainWindow::onTabChange(int index)
+{
+    #if defined(Q_OS_FREEBSD) || defined(Q_OS_OPENBSD) || defined(Q_OS_NETBSD)
```

Comment:
gpassswd[1] is not on bsd. (And no other tool to set group password AFAICT) 
Tab change to groups therefore disables the password button.
[Group passwds are discouraged](https://www.freebsd.org/cgi/man.cgi?query=newgrp&apropos=0&sektion=1&manpath=FreeBSD+11.0-RELEASE&arch=default&format=html)